tools/cli/workflowCommands.go

```diff
@@ -1372,6 +1372,7 @@ func ResetInBatch(c *cli.Context) {
 	// read exclude
 	excludes := map[string]string{}
 	if len(excFileName) > 0 {
+		// #nosec
```

Comment:
@vancexu please review and provide comment.

Reply:
The added comment is correct